src/utils/http.ts

```
@@ -0,0 +1,48 @@
+
+/**
+ * Normalize a pagination token from the API into a URL usable with a client bound to
+ * {@link BASE_URL}. The API returns `next` as an absolute path that already includes the version
+ * prefix (`/api/v3/search/stories?...`), which would otherwise be appended to the base URL and
+ * produce `/api/v3/api/v3/...`.
+ *
+ * Fully qualified URLs are passed through untouched — axios ignores `baseURL` for those.
+ *
+ * @param next - The `next` value returned by a search response
+ */
+export function normalizeNext(next: string): string {
```

ESLint

[error] Declare this without inline export and list it in a single export statement at the end of the file.

(yenz/export-at-end-of-file)